libcontainer/devices/devices_unix.go

```diff
@@ -75,7 +75,7 @@ func getDevices(path string) ([]*configs.Device, error) {
 		switch {
 		case f.IsDir():
 			switch f.Name() {
-			case "pts", "shm", "fd", "mqueue":
+			case "pts", "shm", "fd", "mqueue", ".lxc", ".lxd-mounts":
```

Comment:
You better link the issue as comment, otherwise it'll be hard to understand what they are for normal runc users.